client/my-sites/stats/info-panel/index.jsx

```diff
@@ -200,8 +200,8 @@ class StatsInfoPanel extends React.PureComponent {
 									href="http://wordpress.com/support/stats/#top-posts-pages"
 								>
 									<Gridicon icon="info-outline" />
-									{ this.props.translate( 'About Posts & Pages', {
-										context: 'Stats: Posts & Pages info box documentation link',
+									{ this.props.translate( 'About posts & pages', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.